src/derive.rs

```diff
@@ -47,7 +47,7 @@ use std::ffi::OsString;
 ///         .about("More verbose output"))
 ///     .arg(Arg::new("name")
 ///         .long("name")
-///         .long("n")
+///         .short("n")
```

Comment:
```suggestion
///         .short('n')
```

Reply:
If this is the wrong type, how did this doctest even pass before?